src/ape/managers/converters.py

```diff
@@ -278,7 +280,7 @@ def is_type(self, value: Any, type: Type) -> bool:
         else:
             return isinstance(value, type)
 
-    def convert(self, value: Any, type: Type) -> Any:
+    def convert(self, value: Any, type: Union[Type, Tuple[Type], List[Type]]) -> Any:
```

Comment:
error with pre-commit `(expression has type "Tuple[Any, ...]", variable has type "Tuple[Type[Any]]")`

any suggestions? @unparalleled-js

Reply:
Try `Union[Type, Tuple[Type, ...], List[Type]]

Reply:
`Tuple` wants to know how many, but if you do `...` it says it can be any length.